main.py

```diff
@@ -26,8 +26,123 @@ def nosnotleft():
 elif inp=="Numbers already called yet":
     print(noleft())
 else:
-    print("Invalid Command")
+    print("Invalid Command")'''
+'''CODE FOR BINGO REFERRED FROM YT'''
```

Comment:
Put all this code in a seperate file, remove from here and then send PR. Attach proof of it working